cheatsheets/Choosing_and_Using_Security_Questions_Cheat_Sheet.md

```diff
@@ -1,119 +1,149 @@
 
-### Step 6) Authenticate Requests to Change Questions
+## System Defined Security Questions
 
-Many web sites properly authenticate change password requests simply by requesting the current password along with the desired new password. If the user cannot provide the correct current password, the request to change the password is ignored. The same authentication control should be in place when changing security questions. The user should be required to provide the correct password along with their new security questions & answers. If the user cannot provide the correct password, then the request to change the security questions should be ignored. This control prevents both Cross-Site Request Forgery attacks, as well as changes made by attackers who have taken control over a users workstation or authenticated application session.
+System defined security questions are based on information that is already known about the user. The users' personal details are often used, including the full name, address and date of birth - however these can easily be obtained by an attacker from social media, and as such provide a very weak level of authentication.
+
+The questions that can be used will vary hugely depending on the application, and how much information is already held about the user. When deciding which bits of information may be usable for security questions, the following areas should be considered:
+
+- Will the user be able to remember the answer to the question?
+- Could an attacker easily obtain this information from social media or other sources?
+- Is the answer likely to be the same for a large number of users, or easily guessable?
 
 # Using Security Questions
 
-Requiring users to answer security questions is most frequently done under two quite different scenarios:
+## When to Use Security Questions
+
+Applications should generally use a password along with a second authentication factor (such as an OTP code) to authenticate users. The combination of a password and security questions **does not constitute MFA**, as both factors as the same (i.e, something you know).
+
+**Security questions should never be relied upon as the sole mechanism to authenticate a user**. However, they can provide a useful additional layer of security when other stronger factors are not available. Common cases where they would be use include:
+
+- Authenticating users.
+- Resetting a forgotten password.
+- Resetting a lost MFA token.
+
+### Authentication Flow
+
+Security questions may be used as part of the main authentication flow to supplement passwords. Note that this **does not constitute MFA**, as both factors are the same (i.e, something you know). A typical authentication flow would be:
```

Comment:
Already mentioned the note about `something you know`

Reply:
Fixed in f09ac26.